port/zos390/omrvmem.c

```diff
@@ -166,27 +168,27 @@ get_ipt_ttoken(struct OMRPortLibrary *portLibrary,
 
 /* omrvmem_support_above_bar.s */
```

Comment:
All these comments that refer to omrvmem_support_above_bar.s should be updated to refer to omriarv64.mc

Reply:
Done in 61426a8109918c520868624c078bfa12d22a7564